hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java

```diff
@@ -92,7 +92,7 @@
 
     /** For JSON Construction ONLY */
     public Data() {
-        this(null, 0L, null);
+        this(null, System.currentTimeMillis(), null);
```

Comment:
What am I missing?  This chains to full constructor where we do set to current timestamp if necessary.  We similarly protect in setTimestamp().  Why is this change necessary?